data/members.yml

```diff
@@ -1,490 +1,408 @@
         link: https://siliconassurance.com/
         img_width: 120
 
   - name: partners
     levels: 2
     items:
-       -
-         image: /images/sponsors/amazon-logo.png
-         title: Amazon
-         link: https://aws.amazon.com/
-         img_width: 90
-         level: 1
-       -
-         image: /images/partners/cadence.png
-         title: Cadence
-         link: https://www.cadence.com/
-         img_width: 100
-         level: 1
-       -
-         image: /images/partners/cayenne-global-logo.jpeg
-         title: Cayenne Global, LLC
-         link: https://cayennecom.com/
-         img_width: 120
-         level: 1
-       -
-         image: /images/partners/caspa-logo.jpeg
-         title: Chinese American Semiconductor Professional Association
-         link: http://www.caspa.com/
-         img_width: 120
-         level: 1
-       -
-         image: /images/partners/digilent.png
-         title: Digilent
-         link: https://store.digilentinc.com/
-         img_width: 100
-         level: 1
-       -
-         image: /images/partners/digi-key.png
-         title: DigiKey
-         link: https://www.digikey.com/
-         img_width: 100
-         level: 1
-       -
-         image: /images/partners/eclipse-foundation.svg
-         title: Eclipse Foundation
-         link: https://www.eclipse.org/
-         img_width: 100
-         level: 1
-       -
-         image: /images/partners/fossi-foundation.png
-         title: Fossi Foundation
-         link: https://fossi-foundation.org/
-         img_width: 70
-         level: 1
-       -
-         image: /images/sponsors/openuk-logo.png
-         title: OpenUK
-         link: https://openuk.uk/
-         img_width: 100
-         level: 1
-       -
-         image: /images/partners/orcro.png
-         title: ORCRO
-         link: https://orcro.co.uk/
-         img_width: 100
-         level: 1
-       -
-         image: /images/partners/osfpga-logo.png
-         title: OSFPGA
-         link: https://osfpga.org/
-         img_width: 100
-         level: 1
-       -
-         image: /images/partners/risc.svg
-         title: Risc-V
-         link: https://riscv.org/
-         img_width: 70
-         level: 1
-       -
-         image: /images/partners/bdo.jpg
-         title: BDO
-         link: https://www.bdo.ca/en-ca/home/
-         img_width: 80
-         level: 2
-       -
-         image: /images/partners/moorcrofts.png
-         title: Moorcrofts
-         link: https://moorcrofts.com/
-         img_width: 200
-         level: 2
-       -
-         image: /images/partners/norton-rose-fulbright-logo.svg
-         title: Norton Rose Fulbright
-         link: https://www.nortonrosefulbright.com/en-ca
-         img_width: 200
-         level: 2
-       -
-         image: /images/partners/rbc-logo-shield.svg
-         title: RBC
-         link: https://www.rbc.com/about-rbc.html
-         img_width: 60
-         level: 2
+      - image: /images/partners/amd-xilinx.png
```

Comment:
Amazon comes before AMD in alphabetic order. Small nitpick

Reply:
LOL whoops. Updated